Source/Editor/Viewport/PrefabWindowViewport.cs

```diff
@@ -730,8 +749,7 @@ private void Spawn(Actor actor)
 
         private void Spawn(ScriptType item, SceneGraphNode hit, ref Vector3 hitLocation)
         {
-            var actor = item.CreateInstance() as Actor;
-            if (actor == null)
+            if (!(item.CreateInstance() is Actor actor))
```

Comment:
The previous code is more explicit about object creation and I think it should stay.